osu.Game/Screens/Select/FooterV2/FooterButtonRandomV2.cs

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Input.Events;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Input.Bindings;
+using osuTK;
+using osuTK.Input;
+
+namespace osu.Game.Screens.Select.FooterV2
+{
+    public partial class FooterButtonRandomV2 : FooterButtonV2
+    {
+        public Action NextRandom { get; set; } = null!;
+        public Action PreviousRandom { get; set; } = null!;
```

Comment:
Generally we always make `Action`s `Action?` and support nullability, as a pattern.